### ISSUE TITLE
`introspect:pg` does not have a `--config` option

### ISSUE THREAD

**Issue by remorses** [drizzle-team/drizzle-kit-mirror#61]
Trying to use the config option throws 

```
drizzle-kit introspect:pg --config=drizzle.config.ts
error: unknown option '--config=drizzle.config.ts'
```

**Comment by remorses** [comment]
Also the `--out` path is required while the docs says it defaults to `drizzle`

**Comment by AndriiSherman** [comment]
Thanks! Fixing that

**Comment by AndriiSherman** [comment]
Fixed in `drizzle-kit@0.18.1`
